src/utils.jl

```diff
@@ -40,6 +43,55 @@ function bhattacharyya(x::kmerdist,y::kmerdist)
     return 1/4*log(arg1 + arg2)
 end
 
+"""
+a function that computes the Bhattacharyya coefficient of
+n-distribution
+"""
+function multi_bhattacharyya(data::Array{kmerdist},
+     n::Union{Nothing, Int} = nothing)
+     if n === nothing
+         lensum = 0
+         for d in data
+             lensum += length(d.originaldata)
+         end
+         n = cld(lensum,length(data))
+     end
+     #findes the maximum and minimum values of all the data
+     mini::Float32 = data[1].minVal
+     maxi::Float32 = data[1].maxVal
+     for d in 2:length(data)
+        @inbounds if (mini > data[d].minVal) mini = data[d].minVal end
```

Comment:
I am not sure there is any value in adding @inbounds on top of these if statements. This would make sense on top of for loop. 

@inbounds for d in 2:length(data)